src/modules/crypto/keychainManager.ts

```diff
@@ -107,15 +107,15 @@ const getLocalConfigurationWithoutDB = async (
     const { type } = await get2FAStatusUnauthenticated({ login });
 
     let masterPassword = '';
-    let masterPasswordEnv = process.env.DASHLANE_MASTER_PASSWORD;
```

Comment:
How this has not been catch by the linter ?

Reply:
External contributor, I think the pipelines did not run or something, that was surprising indeed.